rero_ils/modules/loans/utils.py

```diff
@@ -185,6 +177,12 @@ def can_be_requested(loan):
     location = Location.get_record_by_pid(loan.location_pid)
     if not location or not location.get('allow_request'):
         return False
+
+    # 4) Check patron is not blocked
```

Comment:
The order of the execution is important here, I would move the patron blocking to be the first check. If a patron is blocked, I do not need to go and find a cipo (check2) or if he is allowed to request (check3) or even check1

Reply:
You're right. Done